feature/gnoi/system/tests/per_component_reboot_test/per_component_reboot_test.go

```diff
@@ -210,12 +212,21 @@ func TestLinecardReboot(t *testing.T) {
 		}
 		for _, port := range intfsOperStatusUPBeforeReboot {
 			if root.GetInterface(port).GetOperStatus() != oc.Interface_OperStatus_UP {
+				upInterfaces[port] = false
 				return false
+			} else {
+				upInterfaces[port] = true
 			}
 		}
 		return true
 	})
 	if val, ok := watch.Await(t); !ok {
+		for intf, up := range upInterfaces {
+			if !up {
+				gnmi.Get(t, dut, gnmi.OC().Interface(intf).State())
```

Comment:
Hi @mojiiba, what is the purpose of this Get()? The value obtained is not being logged anyway. And if for some reason no value is present at the path, Get() will fail the test fatally thereby skipping the logging at line 227.

Reply:
when we enable -v 2, it will write the interface data in the log that can be used for troubleshooting. note that this get will happen when the test is going to fail, so it will not add a new failure case to the test.

Reply:
Got it, thanks!